loopin.go

```diff
@@ -487,9 +485,9 @@ func (s *loopInSwap) sendUpdate(ctx context.Context) error {
 // execute starts/resumes the swap. It is a thin wrapper around executeSwap to
 // conveniently handle the error case.
 func (s *loopInSwap) execute(mainCtx context.Context,
-	cfg *executeConfig, height int32) error {
+	wg *sync.WaitGroup, cfg *executeConfig, height int32) error {
 
-	defer s.wg.Wait()
+	defer wg.Wait()
```

Comment:
Since the executor will wait for any goroutines started in the swap we don't need this wait.

Reply:
I guess you are referring to this part? https://github.com/lightninglabs/loop/blob/57c9537d49d7ecdae5ade5a76cdb103ede9437d4/executor.go#L219-L222